packages/mdc-textfield/README.md

```diff
@@ -176,6 +176,32 @@ UX for client-side form field validation.
 </p>
 ```
 
+### Leading and Trailing Icons
+Leading and trailing icons can be added to MDC Textfields as visual indicators
+as well as interaction targets. To do so, add the relevant classnames to the root element, add
+an `i` element with your preferred icon, and give it either `mdc-textfield__leading-icon` or
+`mdc-textfield__trailing-icon` depending on which you are looking to achieve.
+
+#### Leading:
+```html
+<div class="mdc-textfield mdc-textfield--box mdc-textfield--with-leading-icon">
+  <i class="material-icons mdc-textfield__leading-icon" tabindex="0">event</i>
```

Comment:
Some clients might not want the icon to be clickable independent of the text field (e.g., a magnifying glass icon on a search field).

Two questions:

1. Should we add a note about removing `tabindex="0"` if the user doesn't want the icon to be clickable?
2. Should we add a bit of CSS that disables pointer events if there's no `tabindex` (or `tabindex="-1"`)? E.g.:
    ```css
    .mdc-textfield__leading-icon:not([tabindex]),
    .mdc-textfield__leading-icon[tabindex="-1"] {
      pointer-events: none;
    }
    ```